lib/models/user.rb

```diff
@@ -4,4 +4,10 @@ class User
   include DataMapper::Resource
 
   property :id, Serial, key: true
-end
\ No newline at end of file
+  property :name, String
+
+  def user
+    User.new
+  end
+  DataMapper.finalize
```

Comment:
Same here. The dataMapper commands are not supposed to be placed in the model but rather in a controller (or extracted to a separate file) Look in /lib/helpers/data_mapper.rb - it's already there.